config/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""Application configuration settings"""
+
+import os
+
+
+class Config(object):
+    """Main configuration class"""
+    DEBUG = False
+    TESTING = False
+    SQLALCHEMY_DATABASE_URI = None
+
+
+class Production(Config):
+    """Production environment configuration"""
+    SQLALCHEMY_DATABASE_URI = os.environ.get('PROD_DATABASE_URL', None)
```

Comment:
If you are planning to deploy with heroku I think this can remain as **DATABASE_URL**  instead of **PROD_DATABASE_URL** since it is provided by default.

Reply:
I would just like to leave it that way from now until the application is ready for deploy